tutorials/base/Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM artemqaapplitools/chrome-docker:latest
```

Comment:
What is the benefit of using a custom chrome docker image instead of https://github.com/SeleniumHQ/docker-selenium ?

Reply:
There are no need in selenium itself, python tutorials use webdriver-manager to download and start chromedriver. Also there are tutorials which uses different tools not based on the selenium.